.babelrc

```diff
@@ -2,8 +2,16 @@
   "presets": [ "es2015-argon" ],
   "sourceMaps": "inline",
   "env": {
-    "test": {
-      "plugins": [ "istanbul" ]
+    "src": {
```

Comment:
generally the "env" is test/prod/dev, not a file location; this suggests that perhaps the production files would ship with the istanbul transform?

perhaps naming these envs "test" and "test-compiled" would be clearer :-)

Reply:
@ljharb Yes very true. Unfortunately `test-compiled` didn't work, it looks like I could not put a dash in the env name. I used `testCompiled` instead: https://github.com/adjerbetian/eslint-plugin-import/commit/a6ed0b3c33e03e6fd0562dbc5c0cc2d447fb5f05